sharktank/tests/export_test.py

```diff
@@ -31,7 +31,7 @@
 
 class ExportTest(TestCase):
     def testFlattenSignature(self):
-        expected_a = [SplitPrimitiveTensor(ts=[torch.tensor([1])], shard_dim=0)]
+        expected_a = [SplitPrimitiveTensor(ts=[torch.tensor([1])], shard_dim=0, devices=(0,), devices_pinned=True)]
```

Comment:
Lets make these tests allow for our default behavior. New tests should check that device placement can be explicitly specified.